core/include/kernel/interrupt.h

```diff
@@ -72,6 +72,8 @@ struct itr_ops {
 	void (*raise_pi)(struct itr_chip *chip, size_t it);
 	void (*raise_sgi)(struct itr_chip *chip, size_t it,
 			  uint32_t cpu_mask);
+	void (*release)(struct itr_chip *chip, size_t it,
```

Comment:
release() is doing other way of add(). so, would it be much clear if they are next to each other?

Reply:
Thanks for your comment, It's sorted them in descending order by the first letter